src/parser/expression/constant_value_expression.cpp

```diff
@@ -250,6 +251,59 @@ std::string ConstantValueExpression::ToString() const {
   }
 }
 
+ConstantValueExpression ConstantValueExpression::FromString(const std::string &val, uint64_t type_id) {
+  const auto cve_type = static_cast<type::TypeId>(type_id);
+
+  switch (cve_type) {
```

Comment:
What about NULL? If you're adding general purpose code to CVE it needs to handle all appropriate use cases, or uncovered uses need to be documented.

Reply:
Then here we can check whether the string is `NULL` and create a NULL CVE using `ConstantValueExpression(const type::TypeId type)`. Double-check with postgres to see whether they accept `null`, `nUlL`, etc.

Reply:
I have added the null checking in the latest push. Let me know if there's any other issues.